test/spinxsk/spinxsk.c

```diff
@@ -2637,7 +3002,18 @@ main(
     ASSERT_FRE(watchdogThread);
 
     //
-    // Kick off the queue workers.
+    // Create global concurrent workers
+    // (non-queue specific operations running concurrently).
+    //
+    if (useFnmp) {
```

Comment:
it would be nice if these threads did not strictly depend on the FNMP switch. maybe make the worker thread itself return immediately if it detects it will never do work?